Dizzy/UI Components/ButtonBuilder.swift

```diff
@@ -11,7 +11,7 @@ import UIKit
 extension UIButton {
     
     var smallRoundedBlackButton: UIButton {
-        let button = UIButton(type: .system)
```

Comment:
if you remove this line an you wont set the button's type there wont be a press effect